### PR TITLE
Fast Ai Documentation 

### PR DESCRIPTION
**Pull request by adricl**

### CODE REVIEW

**Comment by c1505** [comment]
@adricl These docs look good to me . So it is more easily discovered and used by people, what about also adding a link to the training a model page https://docs.donkeycar.com/guide/train_autopilot/ ?

**Comment by adricl** [comment]
I will look at where to add this. Do we have a reviewer for the docs PR?

**Review by c1505 — APPROVED** [review]

**Comment by c1505** [comment]
@adricl I haven't tested it myself yet, but the docs look good to me.  I submitted a review approving it, but I don't have write access so I can't merge.

File: docs/parts/fastai.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+The `FastAILinear` pilot uses one neuron to output a continuous value via the 
+PyTorch Dense layer with linear activation. One each for steering and throttle.
+The output is not bounded.
+
+#### Pros
+
+* Steers smoothly. 
+* It has been very robust.
+* Performs well in a limited compute environment like the Pi3.
```
[the remark this answers is not in the document]
Comment (adricl) [reply docs/parts/fastai.md:22]: I have copied this model info from the tensorflow one. I have not tested it myself. I can change this.

**Comment by adricl** [comment]
@DocGarbanzo @zlite can I please have review of this PR, I think it ready to merge. If you want me to change anything let me know. Cheers

**Comment by adricl** [comment]
I have fixed the merge conflicts now, @DocGarbanzo @zlite is it possible I can have a review. Thanks

**Pull request merged**